### PR TITLE
MohammadH/ Remove gatsby-plugin-page-progress

### PR DESCRIPTION
**Pull request by mohammad-hashemi-deriv**
Changes:

-   This plugin has not been used after removing Academy from deriv-com project

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by NikitK-deriv — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -25,7 +25,6 @@
         "gatsby-plugin-google-tagmanager": "^4.14.0",
         "gatsby-plugin-image": "2.19.0",
         "gatsby-plugin-manifest": "^4.14.0",
-        "gatsby-plugin-page-progress": "^2.2.1",
```
Comment (niloofar-deriv) [inline package.json:28]: are u sure we don't need this?

Comment (niloofar-deriv) [reply package.json:28]: Isn't it the dependency of other plugins?

Comment (mohammad-hashemi-deriv) [reply package.json:28]: To use this plugin in the project we need to add its options to `gatsby-config.js` to enable it and make it work.
At the moment we con't have any configuration for it in `gatsby-config.js`
I remember we used it for academy which is already removed.

Comment (niloofar-deriv) [reply package.json:28]: great :smile:

**Review by niloofar-deriv — APPROVED** [review]

**Pull request closed without merging**